assets/src/dashboard/components/storyAnimation/provider.js

```diff
@@ -105,8 +105,15 @@ function Provider({ animations, children, onWAAPIFinish }) {
     };
   }, []);
 
-  const playWAAPIAnimations = useCallback(() => {
-    WAAPIAnimations.forEach((animation) => animation?.play());
+  const WAAPIAnimationMethods = useMemo(() => {
+    return {
+      play: () => WAAPIAnimations.forEach((animation) => animation?.play()),
+      pause: () => WAAPIAnimations.forEach((animation) => animation?.pause()),
+      setCurrentTime: (time) =>
```

Comment:
Would it make sense to make a `reset` action that internally calls `setCurrentTime(0)` and `pause()`?  This way it'll be clear that we support `reset` (and how we support it), and should there ever come a time when resetting is natively supported by the WAAPI, we'd just have to update our `reset` action and boom, all reset calls around the codebase would be updated.